components/admin_console/billing/plan_details/index.tsx

```diff
@@ -32,13 +31,10 @@ const PlanDetails = ({isFreeTrial, subscriptionPlan}: Props) => {
     const product = useSelector(getSubscriptionProduct);
     const isLegacyFree = useSelector(checkSubscriptionIsLegacyFree);
     const isLegacyFreePaidTier = Boolean(subscription?.is_legacy_cloud_paid_tier);
-    const isCloudFreeEnabled = useSelector(cloudFreeEnabled);
-    let daysLeftOnTrial = getRemainingDaysFromFutureTimestamp(subscription?.trial_end_at);
-    const maxDays = isCloudFreeEnabled ? TrialPeriodDays.TRIAL_30_DAYS : TrialPeriodDays.TRIAL_14_DAYS;
-    if (daysLeftOnTrial > maxDays) {
```

Comment:
this is needed because sometimes it could potentially add one more day to the maximum remaining days, for some minutes/seconds found in the license end date.